controllers/ArticleController.js

```diff
@@ -79,31 +79,47 @@ class ArticleController {
   }
 
   /**
-   * get all articles created
+   * get all articles created and use the query
+   * if provided to implement pagination
    * @param {object} req - request object
    * @param {object} res - response object
    * @param {function} next - next function
    * @returns {object} - the found article from database or empty if not found
   */
   static listAllArticles(req, res, next) {
+    const { page } = req;
+    let limit = null;
+    let offset = null;
+
+    if (page) {
+      // calculate offset and limit
+      limit = 10;
+      offset = limit * (page - 1);
+    }
+
     return Article
       .findAll({
         include: [{
           model: User,
           attributes: { exclude: ['id', 'email', 'hashedPassword', 'createdAt', 'updatedAt'] }
         }],
-        attributes: { exclude: ['userId'] }
+        attributes: { exclude: ['userId'] },
```

Comment:
I thought we agreed to send `userId` with it. Only will not be visible to the user. `userId` links us to other tables.

Reply:
We talked about article `id`, I'll implement the suggestion.